scripts/lib/mkcloud-libvirt.sh

```diff
@@ -57,36 +57,85 @@ function libvirt_net_start()
         ip link set mtu 9000 dev $dev
     done
 
+    boot_mkcloud=/etc/init.d/boot.mkcloud
+    boot_mkcloud_d="$boot_mkcloud.d"
+    boot_mkcloud_d_cloud="$boot_mkcloud_d/$cloud"
+
     if [ -z "$NOSETUPPORTFORWARDING" ] ; then
+        # FIXME: hardcoded assumptions about admin net host range
         nodehostips=$(seq -s ' ' 81 $((80 + $nodenumber)))
```

Comment:
> I would suggest forwarding ports for the admin node and for the controller node's

That's what we already do; not sure what your suggestion is exactly?

> but it should be fine to forward for every node for accessing the other dashboards. In this case would it be better to print a warning somewhere (message/motd) for the same?

Sorry, I've no idea what you mean here.